opennms-base-assembly/src/main/filtered/etc/trapd-configuration.xml

```diff
@@ -1 +1 @@
-<trapd-configuration xmlns="http://xmlns.opennms.org/xsd/config/trapd" snmp-trap-address="*" snmp-trap-port="162" new-suspect-on-trap="false" include-raw-message="false" threads="0" queue-size="10000" batch-size="1000" batch-interval="500"/>
\ No newline at end of file
+<trapd-configuration xmlns="http://xmlns.opennms.org/xsd/config/trapd" snmp-trap-address="*" snmp-trap-port="1162" new-suspect-on-trap="false" include-raw-message="false" threads="0" queue-size="10000" batch-size="1000" batch-interval="500"/>
```

Comment:
Minion also defaults to 1162 - so these will conflict by default when run on the same host now

Reply:
I don't imagine running both on the same system is super common, but doesn't hurt to change it I suppose.